extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -710,11 +709,6 @@ In order to do so remove the existing `submitAdditionalPaymentDetailsResponse` c
 }
 ```
 
-## Step 7: Capture payment (required for Klarna and Affirm)
```

Comment:
What happened to this part?

Reply:
Story: https://github.com/commercetools/commercetools-adyen-integration/pull/848#discussion_r738152731

Basically step 7 describes capture and it has separate docs already so we decided to remove this part. It popped up in this PR because base PR is not up to date with master.